dataset/csv-edge-case-tests/bom-and-data.csv

```diff
@@ -0,0 +1,2 @@
+﻿data,moredata
+abc,﻿﻿bomdata﻿
```

Comment:
We always add a newline at the end of the csv file.

Reply:
It's important to test that files without trailing newlines are also correctly handled.